### DISCUSSION TITLE
Getting an Error in captum_explainer_hetero_link.py file.

### DISCUSSION THREAD

**Discussion by arymandeshwal** [pyg-team/pytorch_geometric#7645] [Q&A] [answered]
Hey guys, I was testing the captum_explainer_hetero_link.py (pytorch_geometric/examples/explain/captum_explainer_hetero_link.py) and I came across an error, I wanted to visualize the explanation and feature importance . I didn't make any changes to the code. I was hoping to use the same approach in my own heterogenous graph and explain it.
It would be amazing if someone could help me out and answer some of my questions.
PS: I'm new to PyG.

The error I got:
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In[16], line 119
    116 print(f'Generated explanations in {explanation.available_explanations}')
    118 path = 'feature_importance.png'
--> 119 explanation.visualize_feature_importance(path, top_k=10)
    120 print(f"Feature importance plot has been saved to '{path}'")

File c:\users\admin'\desktop\course exercises\xai miniproject\.env\lib\site-packages\torch_geometric\data\hetero_data.py:137, in HeteroData.__getattr__(self, key)
    135 elif bool(re.search('_dict$', key)):
    136     return self.collect(key[:-5])
--> 137 raise AttributeError(f"'{self.__class__.__name__}' has no "
    138                      f"attribute '{key}'")

AttributeError: 'HeteroExplanation' has no attribute 'visualize_feature_importance'

**Comment by rusty1s** [comment, marked as answer]
I am sorry, this example currently requires PyG installation from master. We just added the `visualize_feature_importance` functionality for `HeteroExplanation` there.

**Reply by arymandeshwal** [reply]
Oh, that's great. So I guess I just need to reinstall PyG, right?
